tests/FINDOLOGIC/Api/Tests/Definitions/BlockTypeTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace FINDOLOGIC\Api\Tests\Definitions;
+
+use FINDOLOGIC\Api\Definitions\BlockType;
+use FINDOLOGIC\Api\Tests\TestBase;
+
+class BlockTypeTest extends TestBase
+{
+    public function testAllBlockTypesAreAvailable()
+    {
+        $expectedAvailableBlockTypes = [
```

Comment:
These days I tend to abstain from testing static stuff like that but instead focus on testing logic. I see little benefit in it but also little harm.

Reply:
Yeah that was my thought but there is no harm in that, as you already said. Just wanted that cheap code coverage. Also these are just about 5 seconds, so I do not really care in implementing that.